examples/3_write_serial.py

```diff
@@ -43,16 +46,12 @@
     rho.reset_dataset(dataset)
     print("Set the dataset properties for the scalar field rho in iteration 1")
 
-    series.flush()
-    print("File structure has been written")
-
     rho[()] = global_data
 
-    print("Stored the whole Dataset contents as a single chunk, " +
-          "ready to write content")
+    # in Flush_Type.defer, call now:
```

Comment:
The idea is to keep the code valid and compact. Branching here is not be best impression for copy-&-paste examples for my taste, since 100+ lines of code examples are hard to read. Just subjective feeling.